plugins/scaffolder-backend/src/scaffolder/actions/builtin/github/helpers.ts

```diff
@@ -146,7 +146,8 @@ export async function createGithubRepoWithCollaboratorsAndTopics(
           name: repo,
           org: owner,
           private: repoVisibility === 'private',
-          visibility: repoVisibility,
+          visibility:
+            repoVisibility === 'internal' ? undefined : repoVisibility,
```

Comment:
This looks unrelated - was the `internal` value actually not valid for the API?

Reply:
i swear typescript was telling me the api changed but again... too many plates spinning so let me revert and see what's really what.